### ISSUE TITLE
unit test failure: "A Timer is still pending even after the widget tree was disposed."

### ISSUE THREAD

**Issue by wwl901215** [DavBfr/dart_pdf#613]
add this code can fix it:
pdf_preview.dart
```
@override
  void dispose() {
    previewUpdate?.cancel();
    super.dispose();
  }
```

**Comment by DavBfr** [comment]
Can you create a PR?

**Comment by wwl901215** [comment]
I do not have permission发自我的华为手机-------- 原始邮件 --------发件人： David PHAM-VAN ***@***.***>日期： 2021年3月30日周二 晚上8:18收件人： DavBfr/dart_pdf ***@***.***>抄送： wwl901215 ***@***.***>, Author ***@***.***>主    题： Re: [DavBfr/dart_pdf] unit test failure: "A Timer is still pending even after the widget tree was disposed." (#613)
Can you create a PR?

**Comment by DavBfr** [comment]
ok, I'll add it.

**Comment by DavBfr** [comment]
Fixed in printing-5.1.0
